### ISSUE TITLE
Blank Map in Grafana 8.03

### ISSUE THREAD

**Issue by nathant8883** [pR0Ps/grafana-trackmap-panel#65]
Hi, 

I have been using the map plugin successfully across several versions of Grafana. After upgrading to Grafana 8.0.3 the map has stopped working and only shows the default screen:

![image](https://user-images.githubusercontent.com/73909080/124612707-2a0d9480-de38-11eb-947a-0d3ad5ebec9f.png)

My query has not changed since the upgrade and it still shows data when I switch to Grafana stable view. (The data is in Grafana's time-series type)

![image](https://user-images.githubusercontent.com/73909080/124613101-840e5a00-de38-11eb-95b3-48b0f3051a51.png)

Plugin Versions 2.1.2

Any thoughts?

**Comment by CptHolzschnauz** [comment]
Some of the graphs doesn't work anymore (see https://github.com/pR0Ps/grafana-trackmap-panel/issues/67)
The data log thing is one of them. Make another graph (old) with your data and there it works.

**Comment by nathant8883** [comment]
I think you misunderstood the issue. The issue is not about mousing over another panel and getting the markers to show up. The map panel itself is not showing any data / zooming in on it. The table pictured above is the result of the query that is feeding the graph itself.

**Comment by CptHolzschnauz** [comment]
The panel is triggering the points on the map. Some panels don't. Because the log panel you use is not triggering anything on the map you see nothing. Since the last update the trackmap shows the whole globe when there is no data.
That's why i suggested to use an old graph panel with your data to check the issue.

**Comment by feheragoston** [comment]
I also had an empty map after updating Grafana to v8.1.2 and TrackMap to v2.1.2. My datasource is MySQL and I had one query with `time_sec, latitude, longitude`. I had to create separate (A/B) queries `A` with `time_sec, latitude` and `B` with `time_sec, longitude` to make it work again.

**Comment by CptHolzschnauz** [comment]
Working mysql example with a graph (old)
SELECT
  time AS "time",
  B1V AS "Service Battery"
FROM mqtt_msg
WHERE
  $__timeFilter(time) AND device_id= "blbla"
ORDER BY time

**Comment by feheragoston** [comment]
> Working mysql example with a graph (old)
> 
> ```
> SELECT
>   time AS "time",
>   B1V AS "Service Battery"
> FROM mqtt_msg
> WHERE
>   $__timeFilter(time) AND device_id= "blbla"
> ORDER BY time
> ```

I meant the query for the map panel itself. It has nothing to do with other panels, that is a separate issue.

**Comment by CptHolzschnauz** [comment]
so then i don't understand your issue tough. There is no separate query for the map imho (thats the cherry), the table with the data needs a timestamp and a lat/lng to display what happens where. I can confirm it works with Grafana 8.1.2 and mysql.

**Comment by nathant8883** [comment]
@feheragoston I changed my query as you suggested and now some points are showing up, but they are in the middle of Antarctica (bottom center of the panel). Our system is now also running 8.1.0.

![image](https://user-images.githubusercontent.com/73909080/130486410-17384cd0-55a7-41b2-a6dc-b59b16b08b7f.png)

![image](https://user-images.githubusercontent.com/73909080/130486492-e7e9217c-e3aa-4a74-832c-7f36731237b1.png)


@CptHolzschnauz I think you are still confused. The map panel itself needs a query to supply the lat and long to visualize the data points. The picture I posted is the underlying data supplying the map.
